ethsigner/signer/multifile-based/src/test/java/tech/pegasys/ethsigner/signer/multifilebased/KeyPasswordFileTest.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.signer.multifilebased;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static tech.pegasys.ethsigner.signer.multifilebased.KeyPasswordFileFixture.KEY_PASSWORD_1;
+import static tech.pegasys.ethsigner.signer.multifilebased.KeyPasswordFileFixture.KEY_PASSWORD_2;
+import static tech.pegasys.ethsigner.signer.multifilebased.KeyPasswordFileFixture.loadKeyPasswordFile;
+
+import java.nio.file.Path;
+
+import org.junit.jupiter.api.Test;
+
+class KeyPasswordFileTest {
+
```

Comment:
Not part of these changes but KeyPasswordLoader can be private

Reply:
You probably meant this in `KeyPasswordLoaderTest`. I did the change :)

Reply:
Yes that's what I meant :)